core/modules/layout/layout.admin.inc

```diff
@@ -2730,10 +2775,46 @@ function layout_settings_page_validate(&$form, &$form_state) {
  * Submit handler for layout advanced settings page.
  */
 function layout_settings_page_submit($form, &$form_state) {
-  config_set('layout.settings', 'excluded_templates', $form_state['excluded_templates']);
+  // config_set('layout.settings', 'excluded_templates', $form_state['excluded_templates']);
   backdrop_set_message(t('Your configuration has been saved.'));
 }
 
+/**
+ * Given a path, return a list of operations for a group of layouts.
+ */
+function _layout_settings_get_group_operations($template_info, $is_default) {
+  $links = array();
+
+  if ($is_default) {
+    $links['disable'] = array(
+      'title' => t('Disable'),
+      'href' => 'admin/structure/layouts/',
+    );
+  }
+  else {
```

Comment:
This can go at the beginning without the "else". It'll get overwritten by the if statement which is what we want.

Reply:
Not really, the two keys are different ('enable' and 'disable'), so the wouldnt overwrite.